packer/linux/conf/buildkite-agent/hooks/pre-exit

```diff
@@ -2,6 +2,11 @@
 
 set -eu -o pipefail
 
+# clean up our temporary docker config
+if [[ -n "${BUILDKITE_DOCKER_CONFIG_TEMP_DIRECTORY:-}" ]]; then
+  rmdir "$BUILDKITE_DOCKER_CONFIG_TEMP_DIRECTORY"
+fi
```

Comment:
do you think it's worth checking if the directory exists as well? It shouldn't matter, but maybe there's a very small chance a customer build script will have deleted it fist (maybe by cleaning out the system tmp dir or something).

Reply:
Yeah, added a quick commit.

Reply:
oh, this failed in the build too. Looks like it needs a `-rf`

Reply:
interesting - it looks like `rmdir` will only delete empty directories. Maybe we need to use `rm`? Or I guess we could delete the contents first.